### PR TITLE
Pin MSAL version to avoid broken release of MSAL 1.1.16

### PR DESCRIPTION
**Pull request by tjprescott**
Workaround for:
https://github.com/AzureAD/microsoft-authentication-library-for-objc/issues/1280

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -136,6 +136,7 @@
   libraries.
 
 ### Breaking Changes
+
```
Comment (JoshuaLai) [inline CHANGELOG.md:139]: Did you mean to make a change log entry here? @tjprescott

Comment (tjprescott) [reply CHANGELOG.md:139]: Originally yes, just to run the CI and verify it was failing.

Comment (JoshuaLai) [reply CHANGELOG.md:139]: Oh this I assume was just a mistake cause its all the way back in beta.5

**Review by knvsl — APPROVED** [review]

**Comment by JoshuaLai** [comment]
@tjprescott Do we care if we pin it to `1.1.2` as opposed to `1.1.15`. Or is this just telling consumers `15` is the highest we are willing to go?

**Comment by tjprescott** [comment]
The way it is written it will always grab 1.1.15

**Pull request merged**